browser/js/l10n.js

```diff
@@ -132,19 +131,19 @@ var
 	}
 	// Return what function.toLocaleString() normally returns
 	return Function.prototype[$to_locale_string].apply(String_ctr, arguments);
-}
-, loadFromDisk = String_ctr[$to_locale_string] = function (uri) {
-        var fs = require('fs');
-        var read = fs.readFileSync(uri, 'utf8');
+};
+
+const loadFromDisk = function (uri) {

```

Comment:
was: `loadFromDisk = String_ctr[$to_locale_string] = function ....`
`String_ctr[$to_locale_string]` is missing